src/osbuild-manifests/aws_tag.py

```diff
@@ -0,0 +1,79 @@
+import os
+import subprocess
+import json
+import argparse
+
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--stream', dest='stream', type=str, help='Fedora stream', required=True)
+    args = parser.parse_args()
+
+    builds = getBuildsForStream(args.stream)
+    for build in builds:
+        build_id=build['id']
+        arches=build['arches']
+        for arch in arches:
+            print(f"The build is {build_id}")
+            buildFetch(args.stream, build_id, arch)
+            meta = open('builds/'+build_id+'/'+arch+'/meta.json')
+            data = json.load(meta)
+
+            # Delete this when actually running. Just here while I make this script
+            data ={"amis":[{
+                "name": "us-east-1",
+                "hvm": "ami-0016d5df3041499f9",
+                "snapshot": "snap-0c1ca4850fcd5e573"
+            }]}
+            amis = data['amis']
+            for ami in amis:
+                checkAndAddTag(ami["hvm"], ami["name"])
+                checkAndAddTag(ami["snapshot"], ami["name"])
+            return
+
+def checkAndAddTag(resourceId, region):
+    tagExists = checkTag(resourceId)
+    if tagExists:
+        print(f"{resourceId} already tagged with FedoraUser=coreos tag")
+    else:
+        addTag(resourceId, region)
+        print(f"FedoraUser=coreos tag successfully added to {resourceId}")    
+
+def checkTag(resourceId):
+    checkTagCmd = f'aws ec2 describe-tags --filters Name=resource-id,Values={resourceId} Name=value,Values=coreos'

```

Comment:
what does the `Name=value,Values=coreos` do here?

Also do we need to add `--region` to this command too?

Reply:
We don't need `--region` when we are giving --resource-id as an arg.

And for `Name=value,Values=coreos`, that's the way to filter out data where Name is basically "key" and Values is well "value" from tags json data that's spit out from `aws ec2 describe tags`
An example of what I get from this ami is this:
{
    "Tags": [
        {
            "Key": "FedoraUser",
            "ResourceId": "ami-xyx12345",
            "ResourceType": "image",
            "Value": "coreos"
        }
   ]
}